MVCapplication/src/main/webapp/WEB-INF/views/product.jsp

```diff
@@ -8,84 +9,66 @@
   Time: 21:40
 --%>
 
+<style>
+    #edit,#delete {text-align:center}
+</style>
+
 <div class="row">
     <div class="col-lg-12">
         <h1 class="page-header">My products</h1>
     </div>
 </div>
-<!-- /.row -->
+
 <div class="row">
     <div class="col-lg-12">
         <div class="panel panel-default">
-
             <div class="panel-body">
-
-                <button id="addRow">Add new product</button>
-                <button id="deleteRow">Delete product</button>
-
-                    <table width="100%" class="table table-striped table-bordered table-hover" id="productData">
+                <input type="button" onClick="window.location.href = 'addProduct'" value="Add new product">
+                    <table width="100%" class="table table-striped" id="productData">
                         <thead>
                         <tr>
+                            <th>ID</th>
                             <th>Name</th>
                             <th>Description</th>
                             <th>Category</th>
                             <th>Unit</th>
                             <th>Image</th>
                             <th>Stores</th>
+                            <th>Edit</th>
+                            <th>Delete</th>
                         </tr>
                         </thead>
                         <tbody>
                         <c:forEach items="${products}" var="product">
                             <tr class="gradeA">
+                                <td><c:out value="${product.id}"></c:out></td>
                                 <td><c:out value="${product.name}"></c:out></td>
                                 <td><c:out value="${product.description}"></c:out></td>
                                 <td><c:out value="${product.category.name}"></c:out></td>
                                 <td><c:out value="${product.unit.name}"></c:out></td>
-                                <td><c:out value="${product.image}"></c:out></td>
-                                <td><c:out value="${product.image}"></c:out></td>
+                                <td><c:out value="${product.image.id}"></c:out></td>
```

Comment:
`<td><c:out value="${product.image.id}"></c:out></td>`
and this 
`<td>${product.image.id}</td>`
produce the same result in this case